comment/utils.py

```diff
@@ -84,13 +84,31 @@ def is_comment_admin(user):
 
 
 def is_comment_moderator(user):
-    if settings.COMMENT_FLAGS_ALLOWED:
+    if settings.COMMENT_FLAGS_ALLOWED or settings.COMMENT_ALLOW_BLOCKING_USERS:
         return user.groups.filter(name="comment_moderator").exists() or user.has_perm(
             "comment.delete_flagged_comment"
         )
     return False
 
 
+def can_moderate_flagging(user):
+    return bool(
+        settings.COMMENT_FLAGS_ALLOWED and
```

Comment:
i think you can drop the `settings` check here because it is already performed in the `is_comment_moderator` function.